samples/matter/bridge/src/bridge_manager.cpp

```diff
@@ -6,7 +6,23 @@
 
 #include "bridge_manager.h"
 
+#if CONFIG_BRIDGE_ONOFF_LIGHT_BRIDGED_DEVICE
+#include "onoff_light.h"
+#include "onoff_light_data_provider.h"
+#endif
+
+#if CONFIG_BRIDGE_TEMPERATURE_SENSOR_BRIDGED_DEVICE
+#include "temperature_sensor.h"
+#include "temperature_sensor_data_provider.h"
+#endif
+
+#if CONFIG_BRIDGE_HUMIDITY_SENSOR_BRIDGED_DEVICE
+#include "humidity_sensor.h"
+#include "humidity_sensor_data_provider.h"
+#endif
+
```

Comment:
Should bridge_manager.cpp contain the specific implementations? I guess it should be assigned to a data provider somewhere else to leave bridge_manager independent of end implementations.

Reply:
Right and it will be moved out of this module to the dedicated factory module. This is the task that @markaj-nordic is working on.

Reply:
Right, with the factory module we will inject bridged device and data provider objects into the bridge manager.